consensus/service/src/consensus_service.rs

```diff
@@ -704,24 +708,81 @@ impl<E: ConsensusEnclaveProxy, R: RaClient + Send + Sync + 'static> ConsensusSer
 
     /// Helper method for creating the get config json function needed by the GRPC admin service.
     fn create_get_config_json_fn(&self) -> GetConfigJsonFn {
+        let ledger_db = self.ledger_db.clone();
+        let byzantine_ledger = self.byzantine_ledger.clone();
         let config = self.config.clone();
+        let logger = self.logger.clone();
         Arc::new(move || {
+            let mut sync_status = "synced";
+            let mut peer_block_height: u64 = 0;
+            let byzantine_ledger = byzantine_ledger
+                .lock()
+                .expect("Could not get byzantine ledger.");
+            if let Some(byzantine_ledger) = &*byzantine_ledger {
+                if byzantine_ledger.is_behind() {
+                    sync_status = "catchup";
+                };
+                peer_block_height = byzantine_ledger.highest_peer_block();
+            }
+            let block_height;
+            let latest_block_hash;
+            let latest_block_timestamp;
+            let blocks_behind;
+            // If we do not get a num_blocks, several status points will be null
+            match ledger_db.num_blocks() {
+                Ok(b) => {
+                    block_height = Some(b);
+                    latest_block_hash = ledger_db
```

Comment:
I wouldn't call this last block hash, it should be last block id

Reply:
Discussed offline - we're going to go with latest_block_hash to match the request for the endpoint and reduce cognitive load, as the block_id is our name for the block hash, as it functions as the identifier for the block.